cves/CVE-2012-4386.yml

```diff
@@ -119,10 +127,13 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
-  date:
-  automated:
-  apache:
+  answer: |
+    All evidence indicates that the vulnerability was discovered by Rene Gielen;
+    he issued the commit to fix the bug AND created the August third security
```

Comment:
probably unnecessary to capitalize and, but I understand the inflection you want to convey

Reply:
Yeah, it's a bit of an exaggeration, I'll change that.